src/clj/rems/application/model.clj

```diff
@@ -222,7 +222,8 @@
       (assoc :application/forms (vec (:application/forms event)))
       (assoc :application/resources (vec (:application/resources event)))
       (assoc :application/licenses (map #(select-keys % [:license/id])
-                                        (:application/licenses event)))))
+                                        (:application/licenses event)))
+      (assoc-some :application/accepted-licenses {})))
```

Comment:
I think we used to keep track of which licenses you have accepted, so that you would basically only need to accept some new licences. While that is not much of an improvement (to see that yes there are new licenses and must press the same approve licenses -button), in case the licenses required happen to be the same, you don't have to accept again. Have we lost this capability?